api/tests/Api/Users/ReadUserTest.php

```diff
@@ -19,6 +19,18 @@ public function testGetSingleUserIsDeniedForAnonymousUser() {
         ]);
     }
 
+    public function testGetUnknownUserReturns404ForAnonymousUser() {
+        static::createClientWithCredentials()->request('GET', '/users/1');
```

Comment:
"With credentials" means this API call is not anonymous.

Reply:
Sorry, typical case of i will do that quickly before my next appointment. :cry:

Reply:
That's why we do code reviews :)